src/components/QrScan/QrScan.js

```diff
@@ -139,6 +140,8 @@ const QrScan = () => {
   const [scannedData, setScannedData] = useState('');
   const runningQrScanner = useRef(null);
   const scannedCodesRef = useRef([]);
+  // eslint-disable-next-line max-len
+  const { setHealthCardVerified, setHealthCardSupported, setIssuerVerified } = useHealthCardDataContext();
```

Comment:
This would avoid having to disable eslint and would be more consistent with some of the other areas where you expanded objects across multiple lines.
```suggestion
    const {
    setHealthCardVerified,
    setHealthCardSupported,
    setIssuerVerified
  } = useHealthCardDataContext();
```